app/scripts/controllers/network/createJsonRpcClient.js

```diff
@@ -9,17 +9,32 @@ import {
 } from 'eth-json-rpc-middleware';
 import { PollingBlockTracker } from 'eth-block-tracker';
 import { SECOND } from '../../../../shared/constants/time';
+import { calculateAverageBlockTimeInMs } from './calculateAverageBlockTime';
 
 const inTest = process.env.IN_TEST;
 const blockTrackerOpts = inTest ? { pollingInterval: SECOND } : {};
+
+const getChainOpts = async (rpcUrl) => {
+  try {
+    const averageBlockTimeInMs = await calculateAverageBlockTimeInMs(rpcUrl);
+
+    return {
+      pollingInterval: averageBlockTimeInMs * 0.75,
+    };
+  } catch {
+    return {};
+  }
+};
+
 const getTestMiddlewares = () => {
   return inTest ? [createEstimateGasDelayTestMiddleware()] : [];
 };
 
-export default function createJsonRpcClient({ rpcUrl, chainId }) {
+export default async function createJsonRpcClient({ rpcUrl, chainId }) {
```

Comment:
I'm guessing this change to async is causing the CI to fail, any ideas on how to remedy this?